src/ui/profile_preferences.ui

```diff
@@ -156,10 +156,10 @@
           <item>
            <widget class="QCheckBox" name="mRawStreamDump">
             <property name="toolTip">
-             <string>Switch log file format from HTML to raw stream dump in logfiles as sent by server. The logfile can then be can be loaded by Mudlet to test your trigger systems offline.</string>
+             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When checked will cause the date-stamp named log file to be HTML (file extention '.html') which can convey color, font and other formatting information rather than a plain text (file extension '.txt') format.  It is not recommended to change this setting whilst logging is active!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
I'm replying blind from email but IIRC the control variable used to
control HTML/Text is changed directly by the profile preference and if
logging is in progress you will immediately start or stop getting the
HTML tags being added to the on-going log that is controlled by it.
That can't help but make messed up files containing bits with and
without them tags.

Proposed fix: a two control variable solution - one (not controlled by
the user) for the current log file which modifies the behaviour and
which is set/reset at the point the log is started (from the value of)
the other that the user sets/resets as desired...

Will get onto it once I have ~~nailed some of the other jelly to the
ceiling~~taken a look of the script loss issue...

On 12/01/15 20:16, Vadim Peretokin wrote:
> In src/ui/profile_preferences.ui
> <https://github.com/Mudlet/Mudlet/pull/208#discussion-diff-22818444>:
> 
>> @@ -156,10 +156,10 @@
>>            <item>
>>             <widget class="QCheckBox" name="mRawStreamDump">
>>              <property name="toolTip">
>> -             <string>Switch log file format from HTML to raw stream dump in logfiles as sent by server. The logfile can then be can be loaded by Mudlet to test your trigger systems offline.</string>
>> +             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When checked will cause the date-stamp named log file to be HTML (file extention '.html') which can convey color, font and other formatting information rather than a plain text (file extension '.txt') format.  It is not recommended to change this setting whilst logging is active!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
> 
> "It is not recommended to change this setting whilst logging is active!"
> why? This shouldn't do anything while it is active and only take effect
> for the next log.
> 
> People are bound to do something that is possible to do, so someone will
> stuff it up and be upset their logging got stuffed up.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Mudlet/Mudlet/pull/208/files#r22818444>.
>